### DISCUSSION TITLE
Statystyki konta dostępne przez API

### DISCUSSION THREAD

**Discussion by pliszka** [allegro/allegro-api#5341] [Suggestions - resolved]
W tej chwili nie umiem się doszukać w dokumentacji (być może jest) jeśli nie ma to chętnie bym zobaczył możliwość przeglądania statystyk konta jak na allegro:

- https://allegro.pl/moje-allegro/sprzedaz/skutecznosc-wysylki
- https://allegro.pl/moje-allegro/sprzedaz/jakosc-sprzedazy
- https://allegro.pl/moje-allegro/sprzedaz/super-sprzedawca#/index

Myślę, że wszystkim nam zależy na maksymalizowaniu tych statystyk, a dużo łatwiej było by mieć nad nimi pieczę gdyby można było jakoś zautomatyzować proces ich przetwarzania i analizy, jak choćby wykrywać że któreś zaczynają nam nagle spadać, by móc szybciej reagować, aktualnie jedyna opcja to ręczne logowanie się na konta i przeglądanie statystyk co jest dość uciążliwe, bo trzeba to ręcznie porównywać z tym co było wczoraj, przedwczoraj etc.

**Comment by Lukasz-Zurek** [comment]
@pliszka Nie mamy tego typu funkcjonalności w naszym API, dlatego Twój wątek przeniosłem do odpowiedniej zakładki - Discussions, która służy do zamieszczania sugestii.

Dodam, że planujemy wdrożenie funkcjonalności dotyczących jakości sprzedaży w naszym API. Nie mogę jednak zadeklarować konkretnego terminu.

**Comment by tomasz-grzesik** [comment]
@Lukasz-Zurek, wznawiam temat, bo nadal brak tego istotnego dla nas wszystkich zasobu.

**Reply by Lukasz-Zurek** [reply]
@tomasz-grzesik Dopytuję zespołu odpowiedzialnego za funkcjonalności.

**Comment by allegrofan** [comment]
Czy już coś wiadomo w tej sprawie?

**Reply by AureliuszBrussy** [reply]
Wkrótce powinniśmy udostępnić w API statystyki jakości sprzedaży. Będziemy oczywiście to komunikować.

**Comment by PrzemyslawLukanowski** [comment]
Zaktualizowaliśmy status sugestii na "rozwiązania". Statystyki dotyczące jakości sprzedaży [są już od dłuższego czasu dostępne w naszym API](https://developer.allegro.pl/news/sprawdz-jakosc-swojej-sprzedazy-3Ak50n4M3hO).
